linux_sql/sql/queries.sql

```diff
@@ -0,0 +1,25 @@
+SELECT
+    lead(cpu_number, 0) OVER (
+        PARTITION BY cpu_number
+        ORDER BY total_mem DESC
+    ) AS cpu_number,
+    id AS host_id,
+    total_mem
+FROM host_info;
+
+SELECT
+    host_id,
+    hostname,
+    timestamp,
+    avg_used_mem_percentage
+FROM
+    (SELECT
```

Comment:
It would be much easier to have a dedicated method for rounding dates to 5 minutes.